core/app/models/mno_enterprise/user.rb

```diff
@@ -165,6 +165,11 @@ def update_clients!(input)
       process_custom_result(result)
     end
 
+    def clear_clients!
+      result = update_clients({data: {attributes: {set: []}}})
```

Comment:
I'm curious about this line. You're passing `set` as an attribute but the `update_clients` method only permits `add` or `remove`.

Reply:
The `update_clients` method does allow `set` but it only allows it if it is the only attribute. It does not allow it if `add` or `remove` are present.

https://github.com/maestrano/maestrano-hub/blob/develop/app/helpers/api/mnoe/v2/relationship_helper.rb#L19